website/src/components/pageStateSelectors/CompareSideBySidePageStateSelector.tsx

```diff
@@ -23,7 +23,7 @@ export function CompareSideBySidePageStateSelector({
 }) {
     const view = useMemo(() => new Routing(organismsConfig), [organismsConfig]).getOrganismView(organismViewKey);
     const variantFilterConfig = useMemo(
-        () => makeVariantFilterConfig(view.organismConstants, { enableMutationFilter: true }),
+        () => makeVariantFilterConfig(view.organismConstants),
```

Comment:
why is the `{ enableMutationFilter: true }` removed here and below?

Reply:
Ah it seems it is now enabled by default - might be good to add to the PR description as this isn't directly linked to adding advanced queries and it is a larger refactor